### PR TITLE
chore: make twitter and github links to open in new tab

### PR DESCRIPTION
**Pull request by i-naeem**
added `_target=blank` attribute to Twitter and Github link to open those links in new tab.

### CODE REVIEW

File: apps/web/src/components/Navbar.tsx [hunk 1]
```diff
@@ -111,6 +111,7 @@ export const Navbar = () => {
         <ul className="order-2 flex items-center">
           <li>
             <a
+              target="_blank"
```
Comment (ixahmedxi) [inline apps/web/src/components/Navbar.tsx:114]: Please add `rel="noreferrer noopener"` to the elements as well

File: apps/web/src/components/Navbar.tsx [hunk 2]
```diff
@@ -119,6 +120,7 @@ export const Navbar = () => {
           </li>
           <li>
             <a
+              target="_blank"
```
Comment (ixahmedxi) [inline apps/web/src/components/Navbar.tsx:123]: same here

**Review by ixahmedxi — CHANGES_REQUESTED** [review]

File: apps/web/src/components/Navbar.tsx (hunk 1 above)
Comment (i-naeem) [reply apps/web/src/components/Navbar.tsx:114]: I have added that too you can check again :)

File: apps/web/src/components/Navbar.tsx (hunk 2 above)
Comment (i-naeem) [reply apps/web/src/components/Navbar.tsx:124]: yeah same here, I have added the `rel="noreferrer noopener"` too

**Review by ixahmedxi — DISMISSED** [review]

**Review by ixahmedxi — APPROVED** [review]

**Pull request merged**